### DISCUSSION TITLE
Release of version 22.0.6

### DISCUSSION THREAD

**Discussion by jehne** [keycloak/keycloak#25076] [Q&A]
Hello, may I ask, what happened with release of version 22.0.6? I see this version in closed milestones, but it wasn't released yet. Instead of that, version 23.0.0 has been released. Is there still a plan to release version 22.0.6(22.0.7)? thank you

**Comment by rgrosjean** [comment]
Interesting, a tag was created 2 days ago, about a week after the question was asked. Is there a bug fix release coming? There is no docker image for this version.

**Reply by jehne** [reply]
Seems like release process has changed. 23.0.3(tag is on place) version is out and last closed milestone in github is 23.0.0. Also in past, all bug fix releases had corresponding milestone, and this is not true anymore.
